transform/transformers/pck_transformer.py

```diff
@@ -127,6 +130,20 @@ def round_currency_values(self):
                 for k, v in self.data.items()  # noqa
                     if k in self.rsi_currency_questions})  # noqa
 
+    def impute_zero_values(self):
+        """For RSI and QBS Surveys, impute breakdown values as zero if the total
+        provided was zero.
+        """
+        if self.survey.get('survey_id') == self.rsi_survey_id:
+            if 'd20' in self.data:
+                self.data.update({k: '0' for k in self.rsi_turnover_questions})  # noqa
```

Comment:
does it matter if the value is a string or int?

Reply:
All values that come across from eQ are strings, so I though it best to match that